src/cloud/lib/i18n/enUS.ts

```diff
@@ -329,7 +329,8 @@ const enTranslation: TranslationSource = {
   [lngKeys.EditorReconnectAttempt1]: 'Attempting auto-reconnection',
   [lngKeys.EditorReconnectAttempt2]:
     'Changes will not be synced with the server until reconnection',
-  [lngKeys.EditorReconnectDisconnected]: 'Reconnect',
+  [lngKeys.EditorReconnectDisconnected]:
+    'Your edit session has expired. Please click it to reconnect.',
```

Comment:
This is a really long message to be made as a button label, 
it will most likely cause readability issues on small screens ( even my macbook based on the folder depth )

Personally I would keep reconnect as this label, but change the `EditorReconnectDisconnected1` into this.

Reply:
Reverted